semver/gt.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import type { SemVer } from "./types.ts";
+import { compare } from "./compare.ts";
+
+/** Greater than comparison */
+export function gt(
+  s0: SemVer,
+  s1: SemVer,
+): boolean;
+/**
+ * @deprecated (will be removed after 0.195.0) Use `gt(s0: SemVer, s1: SemVer)` instead.
```

Comment:
It's great that the backwards compatibility is still preserver for know - I suggest we give it anywhere between 1-3 months before removing the deprecated API.

So maybe let's be more generous here and deprecate in `0.200.0`?

Reply:
0.200.0 sounds good to me 👍 Updated